### DISCUSSION TITLE
Difference between soketi and k8soketi

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [soketi/pws#837] [Q&A]
What are the differences?

https://artifacthub.io/packages/helm/soketi/soketi

https://artifacthub.io/packages/helm/soketi/k8soketi

Both show compability with kubernetes?

**Comment by rennokki** [comment]
The first helm chart is the one to look for. The other is just an experimental soketi chart for a potential Kubernetes-native API adapter.
